src/containers/Home.js

```diff
@@ -99,7 +99,7 @@ class Home extends Component {
 
     return (
       <div className="authed">
-        <h1>Welcome to Reggie web</h1>
+        <h1>Welcome to Reggie web *** test the difference 3 ***</h1>
```

Comment:
what difference are we testing?

Reply:
that line itself....so I'm just putting some obvious frontend changes to see if the pipeline is rebuilding. I'll remove it.